torch/fx/graph.py

```diff
@@ -284,11 +283,20 @@ class _PyTreeInfo(NamedTuple):
     in_spec: pytree.TreeSpec
     out_spec: Optional[pytree.TreeSpec]
 
+@dataclass(frozen=True)
+class _ParsedStackTrace:
+    """
+    Represents the top-most frame of a parsed stack trace
+    """
+    file: str
+    lineno: str
```

Comment:
Not suer how the lineno is parsed, but should it be typed as an int?

Reply:
It's just interpolated into another string, s'fine ig